.gitignore

```diff
@@ -0,0 +1,41 @@
+# Git ignorance file. Any files in here can be ignored at the root of the
+# directory by prefixing the line with a `/` character. You can explictly not
+# ignore a particular file or directory by prefixing the line with `!`.
+
+# OS and Editor files
+#
+.DS_Store
+*.DS_Store
```

Comment:
I don't actually remember. I've been using the same `.gitignore_global` to make these that I have just kept appending to it over the years.